net/iperf3/src/opnsense/mvc/app/models/OPNsense/iperf/Menu/Menu.xml

```diff
@@ -1,7 +1,7 @@
 <menu>
   <Interfaces>
     <Diagnostics>
-      <iperf3 url="/ui/iperf/" />
+      <iperf url="/ui/iperf/" />
```

Comment:
Uppercase "I", it fits better in the menu, same issue as Rspamd.

Reply:
will do that tomorrow / at the weekend